gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/InMemoryUserQuotaManagerTest.java

```diff
@@ -29,16 +29,16 @@
 import org.testng.annotations.Test;
 
 
-public class UserQuotaManagerTest {
+public class InMemoryUserQuotaManagerTest {
 
-  UserQuotaManager _quotaManager;
+  InMemoryUserQuotaManager _quotaManager;
 
   @BeforeClass
   public void setUp() {
     Config quotaConfig = ConfigFactory.empty()
-        .withValue(UserQuotaManager.PER_USER_QUOTA, ConfigValueFactory.fromAnyRef("user:1,user2:1,user3:1,user6:1"))
-        .withValue(UserQuotaManager.PER_FLOWGROUP_QUOTA, ConfigValueFactory.fromAnyRef("group1:1,group2:2"));
-    this._quotaManager = new UserQuotaManager(quotaConfig);
+        .withValue(InMemoryUserQuotaManager.PER_USER_QUOTA, ConfigValueFactory.fromAnyRef("user:1,user2:1,user3:1,user6:1"))
```

Comment:
what's the difference with AbstractUserQuotaManager.PER_USER_QUOTA?

Reply:
no difference, they are the same shared fields. I changed it to AbstractUserQuotaManager because that's exactly where they are defined.